dapr/clients/http/dapr_actor_http_client.py

```diff
@@ -47,7 +49,14 @@ async def invoke_method(
         """
         url = f'{self._get_base_url(actor_type, actor_id)}/method/{method}'
 
-        body, r = await self._client.send_bytes(method='POST', url=url, data=data)
+        # import to avoid circular dependency
+        from dapr.actor.runtime.reentrancy_context import reentrancy_ctx
```

Comment:
Can you explain how this makes a circular dependency?

Reply:
I honestly can't. When the import was at the module / library level all kinds of tests failed because the wrong things got imported. Doing this import here fixed the issue. The same thing is done in other places in the Python SDK. Basically what is described here: https://stackabuse.com/python-circular-imports

Reply:
IIRC, the actual circular dependency was on dapr.actor.runtime which in turn depends on the http client.